CleanProFinder.Web/src/app/modules/shared/components/card-design/card-design.component.scss

```diff
@@ -0,0 +1,30 @@
+.card-design {
+  border-radius: 40px;
+  height: 100%;
+  aspect-ratio: 630/984;
+  padding: 5%;
+  background-image: url('../../../../../assets/images/card-design-bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  gap: 5%;
+}
+
+.card-design__text {
+    color: #b0f5eb;
+    font-size: 7em;
+    text-align: left;
+    word-break: break-word;
+    word-spacing: 69420px;
+    line-height: 1;
+    font-family: 'Chillax-Semibold';
+}
+
+.card-design__name {
+    color: white;
+    display: block;
+    font-size: 2.5em;
+    text-align: right;
+    font-family: 'GeneralSans-Regular';
```

Comment:
move it to main styles.scss file, use css variables, in general all design specific things must be in one place,